src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -58,20 +58,28 @@
 ;;; their implicit defaults reflect the security issue.  The latter is far more
 ;;; risky for our downstream apps, thus it was decided that it makes sense to
 ;;; keep these overrides.
-(def acceptable-ciphers
-  ["TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"
-   "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"
-   "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
-   "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
+(def tls-1-dot-3-ciphers
+  ["TLS_AES_128_GCM_SHA256"
+   "TLS_AES_256_GCM_SHA384"
+   "TLS_CHACHA20_POLY1305_SHA256"])
+
+(def tls-1-dot-2-ciphers
+  ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
```

Comment:
I think these are in the wrong order?

Reply:
oh, I was looking at Cipher Suite Preferences, point #2 says "The order of cipher suites to be used by preference is:", so I followed that. But I see that that doesn't exactly match the Preferred Cipher Suite Order table.. I'll ask Shellee about it.

Reply:
I think that's saying we should prefer all the 1.3 ciphers before the 1.2 ciphers, then below she lists the order that 1.2 ciphers should be in.

Reply:
yeah, I think you're right, but I'm gonna confirm with Shellee (who is apparently on PTO this week), since it's confusing.

Reply:
okay, looks like what I have is right. According to Diego, we're following Mozilla's recommendations: https://wiki.mozilla.org/Security/Server_Side_TLS